src/studio/nuxeo-online-services-portal/token-management.md

```diff
@@ -38,6 +38,12 @@ For more details on when to use the token, see the <a href="#token-usage">token
 Tokens are only used for command line tools and APIs. You still need to use your password as usual when you log in to Nuxeo Online Services in your browser, for instance when accessing Nuxeo Studio.
 {{/callout}}
 
+{{#> callout type='warning'}}
+The NOS token shouldn't be used to authenticate for packages/hotfixes: 
```

Comment:
I don't understand this part: what are the URLs/pages where the NOS toekn should not be used?